### ISSUE TITLE
about yolov8x-p2 training: not converge

### ISSUE THREAD

**Issue by hd24007** [ultralytics/ultralytics#3099]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

I'm trying to train [yolov8-x-p2] 
https://github.com/ultralytics/ultralytics/blob/main/ultralytics/models/v8/yolov8-p2.yaml

from scratch on COCO2017, but found it can hardly converge in the first 30 epoch. The mAP did not even break 1% at epoch 30, very confused...

4 gpus, total batch=4*4=16, init_lr = 0.01, train_size=640. other params use fefault settings.

then i tried load the backbone of yolov8-x, yet still have little improvement.

I do not require any well pretrained yolov8-p2 weights, but want some suggestions on how to adjust alg-hyps. 

Thank you

**Comment by glenn-jocher** [comment]
@hd24007 P2 model converges correctly. I get 0.01 and 0.05 mAP50-95 on epoch 0 and 1.

```
yolo train model=yolov8n-p2.yaml data=coco.yaml epochs=30 project=p2-test
```

<img width="1162" alt="Screenshot 2023-06-08 at 15 26 01" src="https://github.com/ultralytics/ultralytics/assets/26833433/b2c3f6a7-d93f-4775-8087-2682c4936845">

👋 hi, thanks for letting us know about this possible problem with YOLO 🚀. We've created a few short guidelines below to help users provide what we need in order to start investigating a possible problem.

## How to create a Minimal, Reproducible Example

When asking a question, people will be better able to provide help if you provide **code** that they can easily understand and use to **reproduce** the problem. This is referred to by community members as creating a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example). Your code that reproduces the problem should be:

- ✅ **Minimal** – Use as little code as possible to produce the problem
- ✅ **Complete** – Provide all parts someone else needs to reproduce the problem
- ✅ **Reproducible** – Test the code you're about to provide to make sure it reproduces the problem

For [Ultralytics](https://ultralytics.com/) to provide assistance your code should also be:

- ✅ **Current** – Verify that your code is up-to-date with GitHub with `git pull` or `pip install -U ultralytics` to ensure your problem has not already been solved in the latest code version.
- ✅ **Unmodified** – Your problem must be reproducible using official Ultrlaytics code without changes. [Ultralytics](https://ultralytics.com/) does not provide support for custom code ⚠️.

If you believe your problem meets all the above criteria, please close this issue and raise a new one using the 🐛 **Bug Report** [template](https://github.com/ultralytics/ultralytics/issues/new/choose) with a [minimum reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) to help us better understand and diagnose your problem. 

Thank you! 😃

**Comment by hd24007** [comment]
I found myself did not clear the previous cache. 

I clear the cache, and load model-x's backbone as pretrain for model-x-p2 , the result at epoch 1 looks good now. 

Anyway, thank you very much

**Comment by glenn-jocher** [comment]
@4m7000, we're glad you were able to resolve your issue! Clearing the cache and loading the pre-trained weights can certainly help improve the training accuracy and performance. In the future, if you come across any other questions or issues related to YOLOv8, don't hesitate to ask -- we're always here and happy to help!
